core/src/main/java/org/lflang/diagram/synthesis/LinguaFrancaSynthesis.java

```diff
@@ -129,11 +129,7 @@
 import org.lflang.generator.SendRange;
 import org.lflang.generator.TimerInstance;
 import org.lflang.generator.TriggerInstance;
-import org.lflang.lf.Connection;
-import org.lflang.lf.LfPackage;
-import org.lflang.lf.Model;
-import org.lflang.lf.Reactor;
-import org.lflang.lf.StateVar;
+import org.lflang.lf.*;
```

Comment:
The [style guide](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) forbids these:
```quote
Wildcard imports, static or otherwise, are not used.
```